oauth2client/client.py

```diff
@@ -817,7 +817,8 @@ def _revoke(self, http_request):
       http_request: callable, a callable that matches the method signature of
         httplib2.Http.request, used to make the revoke request.
     """
-    self._do_revoke(http_request, self.refresh_token)
+    token = self.refresh_token or self.access_token
```

Comment:
I don't think this field justifies its existence; just use "self.refresh_token or self.access_token" on the next line.